bears/java/JavaPMDBear.py

```diff
@@ -47,28 +47,28 @@ def create_arguments(filename, file, config_file,
                          allow_unused_code: bool = True):
         """
         :param check_best_practices:
-            Activate to check for best practices.
+            Checks for best practices.
         :param check_braces:
-            Check for the right use of braces.
+            Checks for the right use of braces.
         :param check_clone_implementation:
-            Check for the right implementation of the ``clone()`` function.
+            Checks for the right implementation of the ``clone()`` function.
```

Comment:
IMHO this isn't needed in first place. If it's needed then also mention in commit body that *why* you have made this change because the reason isn't clear here. :D